cmd/validatemodels/validate_models_test.go

```diff
@@ -15,11 +15,12 @@ import (
 
 	"github.com/openfga/openfga/cmd"
 	"github.com/openfga/openfga/cmd/util"
+	"github.com/openfga/openfga/pkg/testutils"
 	"github.com/openfga/openfga/pkg/typesystem"
 )
 
 func TestValidationResult(t *testing.T) {
-	engines := []string{"postgres", "mysql", "sqlite"}
+	engines := append(testutils.PostgresImages, "mysql", "sqlite")
```

Comment:
I'm not a huge fan of this, as its conflating images with engines, but the test suite is missing that abstraction as a whole. I'm open to any architecture suggestions!